src/components/SidePanel.js

```diff
@@ -1,55 +1,56 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import styled from "styled-components";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
-import DataIntake from "./DataIntake";
-import Master from "./Master";
-import DataQC from "./DataQC";
-import Users from "./Users";
+import { BrowserRouter as Router, Link } from "react-router-dom";
 
 const SidePanelIcons = styled.div`
   width: 85px;
   height: 100%;
   background-color: rgb(34, 34, 34);
 `;
 
-class SidePanel extends Component {
-  render() {
+const SidePanel = () => {
+
+  const[showMenu, setShowMenu] = useState(true);
+
+  const handleHideShowMenu = () => {
+    setShowMenu(!showMenu);
+  }
+
+  const SideMenu = () => {
+    return (<div className="side-panel__options flex-column">
+      <div className="client-name">Client Name</div>
+
+      <div>
+        <Link to="/data-intake">

```

Comment:
Currently these Links won't actually function. With `react-router` one needs to specify the component to be rendered when a particular link is visited.